tests/unit/backend/helpers/jest-db.ts

```diff
@@ -222,13 +230,45 @@ const getFullFormById = async (
 ): Promise<IPopulatedForm | null> =>
   await getEmailFormModel(mongoose).getFullFormById(formId)
 
+const insertVerifiedSms = async ({
```

Comment:
i don't think this needs to be a dbHandler function, should it just be exposed as a test function?

Reply:
yep, will change as idt it'll be reused in integration tests as we alreday have service methods to perform such functions. thanks for suggestion!